robocorp-code/bin/create_env/conda.yaml

```diff
@@ -5,8 +5,6 @@ dependencies:
   - python=3.8
   - pip=20.1
   - pip:
+    - robocorp-inspector==0.3.2
     - robotframework
```

Comment:
@fabioz the robotframework is probably not a required dependency for the environment?
I could remove that and let the robocorp-inspector handle its dependencies

Reply:
Actually I think we should keep the `robotframework` dependency because that's also the env that's used by default when the user uses the `Robot Framework Language Server` without using robots.

We should probably pin it to the latest `4.x` version thought...

Reply:
I'll pin it to current latest release ie. robotframework==4.1